### DISCUSSION TITLE
Removing biome with customized worlds

### DISCUSSION THREAD

**Discussion by BiggestBruhMoment** [misode/misode.github.io#511] [Q&A]
So i notice that when im looking at biome setion on customized worlds tools i can only replace one biome with another.Is it possible to just remove the biome? by not filling the entry of what biome i should replace it to??

**Comment by misode** [comment]
The best way to remove a biome is to replace it with a different neighboring biome. Outright removing biomes from the dimension json could lead to ocean biomes spawning on land or vice versa, or strange interactions with the terrain.
